resources/public/api/conf/1.0/schemas/LISATransaction.get.response.schema.json

```diff
@@ -1,86 +1,133 @@
-
-      }
-
-	},
-	"required": [
-		"periodStartDate",
-		"periodEndDate",
-		"inboundPayments",
-        "bonuses"
+        "htbTransferTotalYTD": {
+          "description": "The total amount of Help to Buy funds in the account in the tax year to date.",
+          "type": "number"
+        }
+      },
+      "required": [
+        "htbTransferInForPeriod",
+        "htbTransferTotalYTD"
+      ]
+    },
+    "inboundPayments": {
+      "description": "Details about qualifying deposits and account balance.",
+      "type": "object",
+      "properties": {
+        "newSubsForPeriod": {
+          "description": "The total value of new qualifying deposits paid in to the account during the claim period.",
+          "type": "number"
+        },
+        "newSubsYTD": {
+          "description": "The total value of new qualifying deposits paid in to the account during the tax year to date.",
+          "type": "number"
+        },
+        "totalSubsForPeriod": {
+          "description": "The total amount in the account during the claim period that qualifies for a bonus payment.",
+          "type": "number"
+        },
+        "totalSubsYTD": {
+          "description": "The total amount in the account during the tax year to date that qualifies for a bonus payment.",
+          "type": "number"
+        }
+      },
+      "required": [
+        "newSubsYTD",
+        "totalSubsForPeriod",
+        "totalSubsYTD"
+      ]
+    },
+    "bonuses": {
+      "description": "Bonus payment details.",
+      "type": "object",
+      "properties": {
+        "bonusDueForPeriod": {
+          "description": "The total bonus payment amount due for the claim period.",
+          "type": "number"
+        },
+        "totalBonusDueYTD": {
+          "description": "The total bonus payment amount due for the tax year to date.",
+          "type": "number"
+        },
+        "bonusPaidYTD": {
+          "description": "The total bonus payment that has already been claimed and paid in the tax year to date.",
+          "type": "number"
+        },
+        "claimReason": {
+          "description": "The reason the bonus payment was claimed.",
+          "$ref": "#/definitions/claimReasonType"
+        }
+      },
+      "required": [
+        "bonusDueForPeriod",
+        "totalBonusDueYTD",
+        "claimReason"
+      ]
+    },
+    "supersede": {
+      "description": "Supresede bonus payment by requesting for an additional bonus amount or recover an overpaid amount",
+      "type": "object",
+      "properties": {
+        "automaticRecoveryAmount": {
+          "description": "***TODO****",
+          "type": "number"
+        },
+        "supersededTransactionID": {
+          "description": "***TODO****",
+          "type": "number"
+        },
+        "supersededTransactionAmount": {
+          "description": "***TODO****",
+          "type": "number"
+        },
+        "supersededTransactionResult": {
+          "description": "****TODO****",
```

Comment:
The descriptions exist in the POST, can we use those descriptions here?

Reply:
updated